wagtail/images/models.py

```diff
@@ -412,33 +433,112 @@ def get_rendition(self, filter: Union["Filter", str]) -> "AbstractRendition":
         Note: If using custom image models, an instance of the custom rendition
         model will be returned.
         """
-        if isinstance(filter, str):
-            filter = Filter(spec=filter)
+        if isinstance(filter, Filter):
+            filter_spec = filter.spec
+        else:
+            filter_spec = filter
+        return self.get_renditions(filter_spec)[filter_spec]
+
+    def get_renditions(self, *filter_specs: str) -> Dict[str, "AbstractRendition"]:
+        """
+        Returns a ``dict`` of ``Rendition`` instances with image files reflecting
+        the supplied ``filter_specs``, keyed by the relevant ``filter_spec`` string.
 
+        Note: If using custom image models, instances of the custom rendition
+        model will be returned.
+        """
         Rendition = self.get_rendition_model()
+        filter_map: Dict[str, Filter] = {
+            spec: Filter(spec) for spec in set(filter_specs)
+        }
+        unfulfilled: List[Filter] = list(filter_map.values())
+        return_value: Dict[str, AbstractRendition] = {}
```

Comment:
Nit: `return_value` isn't a very descriptive variable name. Perhaps `found_renditions`?

Reply:
If it's all the same with you, I would like to keep this as is, as I feel it makes things more obvious where we are 'Adding to the return value' later in the code.